waffle_configs/bedrock-prod.env

```diff
@@ -13,44 +13,26 @@ FUNNELCAKE_139_LOCALES=en-US
 FUNNELCAKE_139_PLATFORMS=win
 
 # switches
-SWITCH_BETA_WHATSNEW_68_TRAILHEAD=on
 SWITCH_CONTENTFUL_HOMEPAGE_DE=off
 SWITCH_CONTENTFUL_HOMEPAGE_EN=on
-SWITCH_CONTRIBUTE_SURVEY=off
```

Comment:
Removed in https://github.com/mozilla/bedrock/commit/242f2f1c6d1f3f6c0794533d4b296791682cf425 (actually even before then)